pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -3439,7 +3439,11 @@ public List<String> getOnlineUnTaggedBrokerInstanceList() {
    */
   public List<String> getOnlineUnTaggedServerInstanceList() {
     List<String> instanceList = HelixHelper.getInstancesWithTag(_helixZkManager, Helix.UNTAGGED_SERVER_INSTANCE);
+    List<String> instanceListWithoutTags = HelixHelper.getInstancesWithoutTag(_helixZkManager);
```

Comment:
Suggest adding a method to `HelixHelper` to directly get untagged server instances. Currently we read all instance configs twice

Reply:
Done